model/message.go

```diff
@@ -30,23 +32,12 @@ type Message struct {
 	RoutingKey string
 }
 
-// Fields returns a MapStr holding the transformed message information
-func (m *Message) Fields() map[string]any {
-	if m == nil {
-		return nil
-	}
-	var fields mapStr
-	if m.QueueName != "" {
-		fields.set("queue", map[string]any{"name": m.QueueName})
-	}
-	if m.AgeMillis != nil {
-		fields.set("age", map[string]any{"ms": *m.AgeMillis})
+func (m *Message) toModelJSON(out *modeljson.Message) {
+	*out = modeljson.Message{
+		Body:       m.Body,
+		Headers:    m.Headers,
+		Age:        modeljson.MessageAge{Millis: m.AgeMillis},
+		Queue:      modeljson.MessageQueue{Name: m.QueueName},
+		RoutingKey: m.RoutingKey,
```

Comment:
nit: why is a `toModelJSON` defined for `Message`, where generally it is only defined for event types (error, span, transaction, metricset) and apmEvent and event; doesn't change the functionality but it is a bit inconsistent. 

same question for `Stacktrace`